WebRoot/js/zimbraMail/share/view/ZmAutocompleteListView.js

```diff
@@ -1171,16 +1171,27 @@ function(list, context) {
 	var len = this._matches.length;
 	for (var i = 0; i < len; i++) {
 		var match = this._matches[i];
-		if (match && (match.name || match.email)) {
+		var matchTypeContact = match.type == ZmAutocomplete.AC_TYPE_CONTACT || match.type == ZmAutocomplete.AC_TYPE_GAL;
```

Comment:
For DL's and group, type is set as ZmAutocomplete.AC_TYPE_CONTACT, so it will be handled.